<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>741</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="inputTextEdit"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="formatButton">
         <property name="text">
          <string>格式化</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="compressButton">
         <property name="text">
          <string>压缩</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copyButton">
         <property name="text">
          <string>复制</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exbandButton">
         <property name="text">
          <string>全部展开</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="shrinkButton">
         <property name="text">
          <string>全部收缩</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadFileButton">
         <property name="text">
          <string>选择文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveFileButton">
         <property name="text">
          <string>保存文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="jsonTreeView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJson_Parser">
    <property name="title">
     <string>Json-Parser</string>
    </property>
   </widget>
   <addaction name="menuJson_Parser"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
